cardano-crypto-tests/bls12-381-test-vectors/test-vectors-bls12_381/README.md

```diff
@@ -0,0 +1,58 @@
+## Test vectors for BLS
+### 1- Test vectors for pairing properties
+- `e([a]P, Q) = e(P, [a]Q)`
+- `e([a]P, [b]Q) = e([b]P, [a]Q)`
+- `e([a]P, [b]Q) = e([a * b]P, Q)`
+- `e([a]P, Q) * e([b]P, Q) = e([a + b]P, Q)`
+
+Order of the values printed on `pairing_test_vectors`:
+- `[a]P`    
+- `[b]Q`
+- `[b]P`
+- `[a]Q`
+- `[a + b]P`
+- `[a * b]P`
+
+
+### 2- Test vectors for elliptic curve operations
+Order of the values printed on `ec_operations_test_vectors`:
+- `P \in G_1` 
+- `Q \in G_1`
+- `P + Q \in G_1`
+- `P - Q \in G_1`
+- `[scalar]Q \in G_1`
+- `-P \in G_1`
+- `P \in G_2`
+- `Q \in G_2`
+- `P + Q \in G_2`
+- `P - Q \in G_2`
+- `[scalar]Q \in G_2`
+- `-P \in G_2`
+
+
+### 3- Test vectors for deserialization/decompression
+- Point not in curve should fail deserialisation
+- Point not in group should fail deserialisation
+- Point not in curve should fail decompression
+- Point not in group should fail decompression
+
+Order of the values printed on `serde_test_vectors`:
+- `G1_uncomp_curve`: G1 point uncompressed is on curve?
+- `G1_comp_curve`: G1 point compressed is on curve?
+- `G1_comp_group`: G1 point compressed is on group?
+- `G1_uncomp_group`: G1 point uncompressed is on group?
+- `G2_uncomp_curve`: G2 point uncompressed is on curve?
+- `G2_comp_curve`: G2 point compressed is on curve?
+- `G2_comp_group`: G2 point compressed is on group?
+- `G2_uncomp_group`: G2 point uncompressed is on group?
```

Comment:
I would change the name of these variables from `G1_uncom_curve` into `G1_uncom_not_on_curve`. Also, The question after the semicolon does not seem to provide any extra info.